docs/CONTRIBUTIONS.md

```diff
@@ -0,0 +1,191 @@
+5. Make changes to code and add tests
+6. Update the documentation
+7. Format code
+8. Submit a Pull Request
+
+Each of the steps is detailed below.
+
+### 1. Fork the r5py git repository
+
+Git can be complicated for new users, but you no longer need to use command line
+to work with git. If you are not familiar with git, we recommend using tools on
+GitHub.org, GitHub Desktop or tools with included git like Atom or PyCharm. However, if you
+want to use command line, you can fork r5py repository using following:
+
+    git clone git@github.com:your-user-name/r5py.git r5py-yourname
+    cd r5py-yourname
+    git remote add upstream git://github.com/r5py/r5py.git
+
+This creates the directory r5py-yourname and connects your repository to
+the upstream (main project) r5py repository.
+
+Then simply create a new branch of master branch.
+
+### 2. Create a development environment
+
+A development environment is a virtual space where you can keep an independent
+installation of r5py. This makes it easy to keep both a stable version of
+python in one place you use for work, and a development version (which you may
+break while playing with code) in another.
+
+An easy way to create a r5py development environment is as follows:
+
+- Install [miniconda](http://conda.pydata.org/miniconda.html)
+- Make sure that you have cloned the repository
+- `cd` to the *r5py* source directory
+
+Tell conda to create a new environment, named `r5py`, or any other name you would like
+for this environment, by running::
+
+      conda create -n r5py
+
+This will create the new environment, and not touch any of your existing environments,
+nor any existing python installation.
+
+To work in this environment, you should `activate` it as follows:
+
+      conda activate r5py
+
+You will then see a confirmation message to indicate you are in the new development environment.
+
+To view your environments::
+
+      conda info -e
+
+To return to you home root environment::
+
+      deactivate
+
+See the full conda docs [here](http://conda.pydata.org/docs).
+
+At this point you can easily do a *development* install, as detailed in the next sections.
+
+### 3. Installing Dependencies
+
+To run *r5py* in an development environment, you must first install
+*r5py*'s dependencies. We suggest doing so using the following commands
+(executed after your development environment has been activated)
+to ensure compatibility of all dependencies:
+
+    conda config --env --add channels conda-forge
+    conda config --env --set channel_priority strict
+    conda install configargparse geopandas joblib jpype1 pandas psutil requests pytest pytest-cov codecov requests black
+
+This should install all necessary dependencies including optional and packages for running tests.
+
+### 4. Making a development build
+
+Once dependencies are in place, make an in-place build by navigating to the git
+clone of the *r5py* repository and running::
+
+    python setup.py develop
```

Comment:
Does not work with pep517. 
There’s currently no exact equivalent to it, but the closest would be `pip install --user .` IIRC

Reply:
or to keep it within the environment, probably a simple `pip install .`

Reply:
Yep, I think `pip install .` should do it. Will update accordingly

Reply:
This is fixed.